### ISSUE TITLE
Improve usability of past studies page 

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#648]
# TL;DR
Make it easier to use the past studies page to see what studies your family has done, and to review video from those studies.

# Narrative
As a parent, I would like to be able to see at a glance what studies each of my children has done, and find videos from those studies to remember what happened. Right now, the list is overwhelming and hard to scan; and if I want to look through videos, I have to maximize them to see them, but then minimize & navigate one-at-a-time in order.

# Acceptance Criteria

- [ ] "Cards" representing studies on the past studies page restructured to highlight key info while making it easy to see structure (responses within studies)
- [ ] Videos displayed at a size that's visible without having to make them fullscreen. 
- [ ] Video navigation allows navigating forward/back (e.g. with carousel selector, small pictures that can be clicked to enlarge, or just standard << 1 ... 5 ... end >> type navigation)
- [ ] Past responses paginated (may be paginated by study rather than response, depending on structure)
- [ ] Tabs per child (or other way of sorting by child), with first child's results loaded by default and others deferred
- [ ] Tests that appropriate studies are displayed in list
- [ ] QA on staging
